install/src/main/java/org/jetbrains/bsp/bazel/install/Install.kt

```diff
@@ -55,6 +56,12 @@ object Install {
         return environmentCreator.create()
     }
 
+    private fun createEnvironmentOutside(details: BspConnectionDetails, cliOptions: CliOptions): Try<Void> {
```

Comment:
what is the purpose of this method?

Reply:
it is creating an environment for provided Bazel workspace root dir

Reply:
I believe that the existing mechanism could be still used